ubtue_Wiley Online Library.js

```diff
@@ -381,7 +384,7 @@ function scrapeBibTeX(doc, url) {
 			addFreeAccessTag(doc, item);
 			getORCID(doc, item);
 			normalizeIssue(doc, item);
-			item.title = item.title.replace(/☆+$/, '');
+			item.title = item.title.replace(/\*+$/, '');
```

Comment:
It seems better to move this functionality to a function, i.e. `normalizeTitle()` and if possible call only at one place.